.github/workflows/test-bot.yml

```diff
@@ -0,0 +1,23 @@
+name: brew test-bot
+on:
+  pull_request:
+    paths:
+      - "Formula/govuk-connect.rb"
+  push:
+    paths:
+      - "Formula/govuk-connect.rb"
```

Comment:
I feel like it'd be worth to leave a comment as to why we're only doing this for govuk-connect. You might also want to use a yaml alias to share the array if the comment ends up bulky.